src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -1022,51 +1049,22 @@ class TestLibrary(SystemTests):
     def session_creation_kwargs(self):
         return {}
 
-    # Test doesn't run over gRPC due to issues with the name of the property attribute_value.
```

Comment:
This seems wrong, but the d305 metadata introduced this test failure for some reason.

Reply:
Talked more with @DavidCurtiss  about this. The function was very recently deprecated and its replacement isn't fully supported in the driver yet, so we're not sure what the best immediate course of action is. Need to discuss this with Jay when he's back in office next week.

Reply:
I removed a couple of functions from grpc metadata generation since the driver doesn't fully support them yet, and then I re-enabled a couple of functions that were incorrectly marked deprecated.